kinetic/release-armhf-build.yaml

```diff
@@ -0,0 +1,120 @@
+%YAML 1.1
+# ROS buildfarm release-build file
+---
+abi_incompatibility_assumed: true
+jenkins_binary_job_priority: 94
+jenkins_binary_job_timeout: 600
+jenkins_source_job_priority: 79
+jenkins_source_job_timeout: 30
+notifications:
+  emails:
+  - ros-buildfarm-kinetic@googlegroups.com
+  - esteve+buildfarm@osrfoundation.org
+  maintainers: false
+package_blacklist:
+  - ardrone_autonomy
+  - avt_vimba_camera
+  - baxter_ikfast_left_arm_plugin
+  - baxter_ikfast_right_arm_plugin
+  - bride
+  - epos_library
+  - eus_nlopt
+  - eus_qp
+  - gazebo_ros
+  - graspit
+  - h4r_x52_joyext
+  - homer_nav_libs
+  - icar_mini
+  - imagesift
+  - jaco_sdk
+  - jinteractiveworld
+  - jsk_pcl_ros
+  - kdl_typekit
+  - libfovis
+  - libntcan
+  - libpointmatcher
+  - libsegwayrmp
+  - libvimba
+  - map_merger
+  - mapviz
+  - micros_rtt
+  - multires_image
+  - nao_meshes
+  - naoqi_driver
+  - nav_pcontroller
+  - ndt_rviz_visualisation
+  - neo_relayboard
+  - nerian_sp1
+  - octovis
+  - oculus_sdk
+  - ola_ros
+  - openhrp3
+  - or_libs
+  - p2os_urdf
+  - pcan_topics
+  - pepper_meshes
+  - pointgrey_camera_driver
+  - pr2_motor_diagnostic_tool
+  - prosilica_gige_sdk
+  - qglv_gallery
+  - qglv_opengl
+  - riskrrt
+  - robot_calibration
+  - robot_face
+  - rosjava_bootstrap
+  - rospeex_core
+  - rostful_node
+  - sr_external_dependencies
+  - teb_local_planner
+  - ueye_cam
+  - uwsim
+  - uwsim_osgocean
+  - uwsim_osgworks
+  - visp
```

Comment:
Should we copy the list from jade or start with an empty list and add packages when they have been released and fail?